### PR TITLE
Recognize known html tags

### PR DESCRIPTION
**Pull request by fisker**
- [x] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: tests/html/tags/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -3764,6 +3764,195 @@ printWidth: 80
...
+
+=====================================output=====================================
+<noscript>noscript</noscript
+><details><summary>summary long long long long </summary>details</details
+><dialog open>dialog</dialog
+><object data="horse.wav"
+  ><param name="autoplay" value="true" /><param
+    name="autoplay"
+    value="true" /></object
+
```
Comment (fisker) [inline tests/html/tags/__snapshots__/jsfmt.spec.js.snap:3825]: This should be a bug.

Comment (fisker) [reply tests/html/tags/__snapshots__/jsfmt.spec.js.snap:3825]: #8626

File: src/language-html/constants.evaluate.js [hunk 2]
```diff
@@ -23,16 +23,23 @@ const CSS_DISPLAY_TAGS = {
...
-
   button: "inline-block",
 
   // special cases for some css display=none elements
   template: "inline",
   source: "block",
   track: "block",
   script: "block",
+  param: "block",
+  noscript: "none",
```
Comment (thorn0) [inline src/language-html/constants.evaluate.js:34]: Why `none`? How is it different from `block`?

Comment (fisker) [reply src/language-html/constants.evaluate.js:34]: I  need  update tests, current example not good.

Comment (fisker) [reply src/language-html/constants.evaluate.js]: `none` seems same as `inline`, I'll check if `noscript` can set to `block`

Comment (fisker) [reply src/language-html/constants.evaluate.js]: On chrome, `<noscript>` is `inline`

image

**Comment by fisker** [comment]
Forgot snapshot outdated, fixing.

**Pull request merged**